Monika After Story/game/script-farewells.rpy

```diff
@@ -1460,27 +1460,110 @@ init 5 python:
     )
 
 label bye_prompt_housework:
-    if mas_isMoniNormal(higher=True):
-        m 1eub "Doing your chores, [player]?"
-        m 1ekc "I would like to help you out, but there's not really much I can do since I'm stuck in here..."
-        m 3eka "Just make sure to come back as soon as you're done, okay?"
-        m 3hub "I'll be waiting here for you~"
-    elif mas_isMoniUpset():
-        m 2esc "Fine."
-        m 2tsc "At least you're doing something responsible."
-        m 2tfc "{cps=*2}...For once.{/cps}{nw}"
+#TODOS: Chance for upset/distressed to not want to go with you, if not I'd ideally like it to default to regular goodbye
+# How to set it to non-generic "we're home" greeting?
+    if persistent.mas_carryme_choice = mas.dockstat.CM_TAKE_MONI
```

Comment:
We can also likely clean up the checks a little by reordering.

Basically, if we check `if persistent._mas_carryme_choice not in [mas_dockstat.CM_LEAVE_MONI, mas_dockstat.CM_WAIT_MONI, None]`, we know the selection is either automatically take her with, or have her ask. Combine this with the affection check and we can keep the existing dialogue first for cleanliness.